src/titlepane/tests/unit/TitlePane.ts

```diff
@@ -0,0 +1,208 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import harness, { Harness } from '@dojo/test-extras/harness';
+import { compareProperty } from '@dojo/test-extras/support/d';
+import { v } from '@dojo/widget-core/d';
+import TitlePane, { TitlePaneProperties } from '../../TitlePane';
+import * as css from '../../styles/titlePane.m.css';
+import { Keys } from '../../../common/util';
+
+const isNonEmptyString = compareProperty((value: any) => {
+	return typeof value === 'string' && value.length > 0;
+});
+
+let titlePane: Harness<TitlePaneProperties, typeof TitlePane>;
+registerSuite({
+	name: 'TitlePane',
+
+	beforeEach() {
+		titlePane = harness(TitlePane);
+	},
+
+	afterEach() {
+		titlePane.destroy();
+	},
+
+	'default rendering'() {
+		titlePane.setProperties({
+			title: 'test'
+		});
+
+		titlePane.expectRender(v('div', {
+			classes: titlePane.classes(css.root)
+		}, [
+			v('div', {
+				'aria-level': '',
+				classes: titlePane.classes(css.title, css.closeable),
+				onclick: titlePane.listener,
+				onkeyup: titlePane.listener,
+				role: 'heading'
+			}, [
+				v('div', <any> {
```

Comment:
I got the same comment on Calendar tests, and I think it may be better to individually cast the diff functions instead of the entire properties object.